FakeNFT/Scenes /Profile/MyNFT/MyNFTCell.swift

```diff
@@ -0,0 +1,242 @@
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        contentView.isUserInteractionEnabled = true
+        selectionStyle = .none
+        setupViews()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: - Public Methods
+    func configure(nft: Nft) {
+        
+        nftImageView.kf.setImage(with: nft.images[0], placeholder: UIImage(systemName: "photo"))
+        nftNameLabel.text = nft.name
+        nftAuthorLabel.text = nft.author
+        nftPriceLabel.text = "\(nft.price) ETH"
+        
+        let clampedRating = max(0, min(nft.rating, 5))
+        
+        for (index, star) in starImageViews.enumerated() {
+            star.image = UIImage(systemName: "star.fill")
+            star.tintColor = index < clampedRating ? .systemYellow : .lightGray
+        }
+    }
+    
+    // MARK: - Private Methods
+    private func setupViews() {
+        contentView.addSubview(mainStackView)
+        
+        mainContentStackView.addArrangedSubview(nameAndRatingStackView)
+        mainContentStackView.addArrangedSubview(priceStackView)
+        
+        nameAndRatingStackView.addArrangedSubview(nftNameLabel)
+        nameAndRatingStackView.addArrangedSubview(ratingStarsStackView)
+        nameAndRatingStackView.addArrangedSubview(authorDetailsStackView)
+        
+        authorDetailsStackView.addArrangedSubview(nftFromAuthorLabel)
+        authorDetailsStackView.addArrangedSubview(nftAuthorLabel)
+        
+        priceStackView.addArrangedSubview(nftPriceTitleLabel)
+        priceStackView.addArrangedSubview(nftPriceLabel)
+        
+        for _ in 0..<5 {
+            let starImageView = UIImageView()
+            starImageView.contentMode = .scaleAspectFit
+            ratingStarsStackView.addArrangedSubview(starImageView)
+            starImageViews.append(starImageView)
+            
+            starImageView.translatesAutoresizingMaskIntoConstraints = false
+            NSLayoutConstraint.activate([
+                starImageView.heightAnchor.constraint(equalToConstant: 12),
+                starImageView.widthAnchor.constraint(equalToConstant: 12)
+            ])
+        }
+        
+        NSLayoutConstraint.activate([
+            mainStackView.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 16),
+            mainStackView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -16),
+            mainStackView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 16),
+            mainStackView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -39),
+            
+            ratingStarsStackView.heightAnchor.constraint(equalToConstant: 12),
+        ])
+    }
+    
+    // MARK: - Actions
+    @objc private func heartButtonTapped() {
+        if heartButton.tintColor == UIColor.white {
```

Comment:
Можно записать одной строчкой: 
```suggestion
     heartButton.tintColor = (heartButton.tintColor == UIColor.white) ? UIColor.red : UIColor.white
```